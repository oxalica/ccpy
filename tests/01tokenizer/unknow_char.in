# Unknow symbol
1 2
3$
