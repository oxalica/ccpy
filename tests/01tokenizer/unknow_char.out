Integer("1")
Integer("2")
Newline
Integer("3")
Fail: Invalid token beginning
